doc/classes/Time.xml

```diff
@@ -180,7 +180,7 @@
 		<method name="get_unix_time_from_system" qualifiers="const">
 			<return type="float" />
 			<description>
-				Returns the current Unix timestamp in seconds based on the system time in UTC. This method is implemented by the operating system and always returns the time in UTC.
+				Returns the current Unix timestamp in seconds based on the system time. The Unix timestamp is the number of seconds passed since 00:00:00 UTC on 1 January 1970, the <link title="Unix epoch">https://en.wikipedia.org/wiki/Unix_time</link>. It is always in UTC.
```

Comment:
```suggestion
				Returns the current Unix timestamp in seconds based on the system time. The Unix timestamp is the number of seconds passed since 00:00:00 UTC on 1 January 1970, the [url=https://en.wikipedia.org/wiki/Unix_time]Unix epoch[/url]. It is always in UTC.
```